UIViewController+Alert.m

```diff
@@ -32,6 +32,20 @@ - (void)showActionSheetWithTitle:(NSString *)title message:(NSString *)message o
     [self showActionSheetWithTitle:title message:message options:options handlers:handlers presentFromView:nil];
 }
 
+- (void)showiPadActionSheetWithTitle:(NSString *)title message:(NSString *)message options:(NSArray *)options handlers:(NSArray *)handlers presentFromTable:(UITableView *)presentingTable withIndexPath:(NSIndexPath *)indexPath {
+    CGRect cellRect = [presentingTable cellForRowAtIndexPath:indexPath].frame;
+    cellRect.origin.y += cellRect.size.height * 2;
+    cellRect.origin.y -= presentingTable.contentOffset.y;
+    cellRect.size.height = 1;
+    
+    [UIActionSheet showFromRect:cellRect inView:self.view animated:YES withTitle:title cancelButtonTitle:@"Cancel" destructiveButtonTitle:nil otherButtonTitles:options tapBlock:^(UIActionSheet *actionSheet, NSInteger buttonIndex) {
```

Comment:
Also see [https://github.com/project-badass/UIViewController-Alert/pull/1/files#diff-60e01a7d1ce88bad5e5cbbb673a45e67R75](here) to use a `UIPopoverPresentationController` if the user is on iPad